src/openforms/contrib/brk/validators.py

```diff
@@ -21,6 +21,8 @@
 
 logger = logging.getLogger(__name__)
 
+BRK_BUSINESS_VALIDATOR = "brk-zakelijk-gerechtigd"
```

Comment:
this is such a highly specific term, I don't think it's wise to attempt to translate it

```suggestion
BRK_ZAKELIJK_GERECHTIGD_VALIDATOR_ID = "brk-zakelijk-gerechtigd"
```